### PR TITLE
Removed job name from template file due to pipeline error

### PR DESCRIPTION
**Pull request by endakelly**

### CODE REVIEW

**Review by timja — COMMENTED** [review]
doesn't seem the right fix according to docs:
https://docs.microsoft.com/en-gb/azure/devops/pipelines/process/phases?tabs=yaml&view=azure-devops

**Comment by endakelly** [comment]
> doesn't seem the right fix according to docs:
> https://docs.microsoft.com/en-gb/azure/devops/pipelines/process/phases?tabs=yaml&view=azure-devops

I was going by this section in this doc https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#job-reuse "When working with multiple jobs, remember to remove the name of the job in the template file, so as to avoid conflict" which seems to match the issue

**Review by timja — APPROVED** [review]

File: builds/acr-cleanup/template-image-cleanup.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ parameters:
   acrResourceGroup: ''
 
 jobs:
-- job: ACR_Cleanup
+- job:
   pool:
     vmImage: 'Ubuntu 16.04'
```
Comment (timja) [inline builds/acr-cleanup/template-image-cleanup.yaml:10]: change this to latest please

Comment (endakelly) [reply builds/acr-cleanup/template-image-cleanup.yaml]: updated

**Pull request merged**